lib/src/builder/parsing/members/props_and_state.dart

```diff
@@ -110,15 +110,13 @@ abstract class BoilerplatePropsOrState extends BoilerplateTypedMapMember
         break;
       case Version.v3_legacyDart2Only:
         _sharedLegacyValidation(errorCollector);
-        if (node is ClassOrMixinDeclaration) {
```

Comment:
The `checkForMetaPresence` case ends up being a no-op for the `ClassTypeAlias` case since it doesn't have any members, so this code is functionally equivalent to how it was before.

Since `checkForMetaPresence` now takes a `ClassishDeclaration` instead of the class node, it seemed awkward and unnecessary to perform that check here.